plugins/Goals/lang/en.json

```
@@ -110,9 +110,9 @@
         "Products": "Products",
         "ProductSKU": "Product SKU",
         "ProductSKUs": "Product SKUs",
-        "ReturningVisitorsConversionRateIs": "Returning visitors conversion rate is %s",
+        "ReturningVisitorsConversionRateIs": "The conversion rate for returning visitors is %s",
         "SingleGoalOverviewDocumentation": "This is an overview of the conversions for a single goal. %s The sparklines below the graph can be enlarged by clicking on them.",
-        "ThereIsNoGoalToManage": "There is no goal to manage for website %s",
+        "ThereIsNoGoalToManage": "There is no goal to manage for the %s website",
```

The placeholder contains the name of a website...
```suggestion
        "ThereIsNoGoalToManage": "There is no goal to manage for website %s",
```

---

That makes sense in German, but not in English.

---

Might be the reason why that feels wrong to me 🙈 
Nevertheless we could consider adding quotes here as well. Wondering if it otherwise might look ugly in English if someone names his site with multiple words.